jetbrains-rider/src/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetRuntimeGroup.kt

```diff
@@ -16,7 +16,8 @@ class DotNetRuntimeGroup : SdkBasedRuntimeGroupInformation() {
     override val runtimes: Set<Runtime>
         get() = setOf(
                 Runtime.DOTNETCORE2_0,
```

Comment:
shouldn't this one be removed?

Reply:
We still allow run/debug of deprecated runtimes today